api/app/Policies/PoolPolicy.php

```diff
@@ -172,7 +172,7 @@ public function updatePublished(User $user, Pool $pool)
      */
     public function publish(User $user, Pool $pool)
     {
-        if (! $pool->getStatusAttribute() === PoolStatus::DRAFT->name) {
+        if (! ($pool->getStatusAttribute() === PoolStatus::DRAFT->name)) {
             return Response::deny('Pool has already been published.');
         }
 
```

Comment:
An actual bug found! This would always evaluate to false before as it would evaluate it as
`( ! $pool->getStatusAttribute()) === PoolStatus::DRAFT->name`

Note the parentheses

Reply:
We could even simplify this further if we want to make it not so confusing :sweat_smile: 

```php
if($pool->status !== PoolStatus::DRAFT->name) {}
```

Reply:
Yeah, I think that looks better https://github.com/GCTC-NTGC/gc-digital-talent/pull/11666/commits/929b5acc0af4c8c846c15e1e740349f4cb923e8a

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11666/commits/da0eb9553261d013ed08d80830617af7c6720fac